<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverflowWidget</class>
 <widget class="QWidget" name="OverflowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#settingsLayout {
 padding: 10px;
 border-style: solid;
 border-width: 3px;
 border-radius: 7px;
}</string>
  </property>
  <widget class="QPushButton" name="cleanButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>111</width>
     <height>59</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>You can cancel </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>them pressing </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>&quot;Clean&quot; button</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="newSettingsButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>New Settings</string>
   </property>
  </widget>
  <widget class="QWidget" name="emptyWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#emptyWidget {
 padding: 10px;
 border-style: solid;
 border-width: 3px;
 border-radius: 7px;
}</string>
   </property>
   <widget class="QLabel" name="emptyLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#emptyLabel {
color: grey;
font-size: 13pt;
}</string>
    </property>
    <property name="text">
     <string>No preferences chosen</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>289</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>This is an overflow management system</string>
   </property>
  </widget>
  <widget class="QWidget" name="fullWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="settingsLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>SUPPLEMENTARY CARD:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="cardLabel">
       <property name="text">
        <string>&lt;Card&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>RECEIVER:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>&lt;Receiver&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>FUNDS OVER:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="maxSumLabel">
       <property name="text">
        <string>&lt;Max Sum&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_2 {
text-decoration:underline;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>Your current overflow settings:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
